### PR TITLE
move to kustomize based deployment

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -12,9 +12,11 @@
 
 ## Deploy the Operator
 
-Deploy the operator by running the following command.
+Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
+with the desired [release tag](https://github.com/confidential-containers/operato/tags).
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```
Comment (fidencio) [inline docs/INSTALL.md:19]: Users will want to just copy and paste the instructions.
Would be possible to have <RELEASE_VERSION> as something automagically populated with the latest release?

File: docs/INSTALL.md [hunk 2]
```diff
@@ -141,5 +143,5 @@ kubectl delete  -f https://raw.githubusercontent.com/confidential-containers/ope
 
 ### Delete the Operator
 ```
-kubectl delete -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl delete -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```
Comment (fidencio) [inline docs/INSTALL.md:146]: Same comment here.

File: docs/INSTALL.md (hunk 1 above)
Comment (fidencio) [reply docs/INSTALL.md:19]: I take this back, this is not that simple to do without assuming users have some specific tools installed on their OSes.

File: docs/INSTALL.md (hunk 2 above)
Comment (fidencio) [reply docs/INSTALL.md:146]: I take this back, this is not that simple to do without assuming users have some specific tools installed on their OSes.

**Comment by fidencio** [comment]
We have a chicken-egg problem with this PR and https://github.com/confidential-containers/operator/pull/123.

We need a newer payload of Kata Containers in order to have the tests passing, but we can only provide a newer payload together with the kustomize change provided by this PR.

What I'd like to suggest is that we review this one normally, but don't merge it.  After we agree on what should be merged, I'll pull the changes as part of https://github.com/confidential-containers/operator/pull/123.

Makes sense?

File: docs/INSTALL.md [hunk 3]
```diff
@@ -12,9 +12,11 @@
 
 ## Deploy the Operator
 
-Deploy the operator by running the following command.
+Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
+with the desired [release tag](https://github.com/confidential-containers/operator/tags).
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```
Comment (bpradipt) [inline docs/INSTALL.md:19]: Can we just add a concrete example using v0.2.0 as the RELEASE_VERSION ? It'll be helpful imho ?

**Comment by fidencio** [comment]
This one was merged as part of #123

**Pull request closed without merging**